internal/db/schema/migrations/generate/generate.go

```diff
@@ -96,9 +90,23 @@ func generate(dialect string) {
 			if fullV > largestSchemaVersion {
 				largestSchemaVersion = fullV
 			}
+
+			cbts, err := ioutil.ReadFile(fmt.Sprintf("%s/%s/%s/%s", srcDir, dialect, ver, name))
+			if err != nil {
+				fmt.Printf("error opening file %s with dialect %s: %v", name, dialect, err)
+				os.Exit(1)
+			}
+
+			contents := strings.TrimSpace(string(cbts))
```

Comment:
An alternate way to do this which might be a tiny bit "safer" (in case anyone does "Begin" or something) would be to split the lines, then strip out the first line if its lower-case equivalent contains "begin", and do the same thing with end, then reconstruct it with newlines.

Either way works, I think it's unlikely anyone will do mixed case and it will be caught immediately if they do.

Reply:
Please take a look at my new approach where I just compare the lowercase version of the first len("begin;") characters and the last len("commit;") characters.